.github/workflows/test.yml

```diff
@@ -0,0 +1,45 @@
+name: Test rq
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+    name: Python${{ matrix.python-version }}/Redis${{ matrix.redis-version }}/redis-py${{ matrix.redis-py-version }}
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [3.5, 3.6, 3.7, 3.8]
+        redis-version: [3, 4, 5, 6]
+        redis-py-version: [3.5.0, 3.5.3]
```

Comment:
Do we need to test for two redis-py versions for now? If not, we can keep the list syntax but only put 3.5.3.

Reply:
Why not? If we don't test 3.5.0 than we should set the minimum required `redis-py` version to 3.5.3.